backend/test/api/effects_test.clj

```diff
@@ -0,0 +1,23 @@
+(ns api.effects-test
+  (:require [expectations.clojure.test :refer :all]
+            [clojure.test :as ctest]
+            [mocking :as mocking]
+            [api.base :as api]))
+
+(ctest/use-fixtures :each mocking/mock-persistence)
+
+(defexpect playing-effects
```

Comment:
I'm not sure this test is needed, but better to be safe than sorry.

Reply:
The test is fine for now, although it's the kind of test I'm wanting to simplify with #98